src/app/grants/page.tsx

```diff
@@ -0,0 +1,194 @@
+    date: new Date('2024-10-01'),
+    delegateTo: '0x01',
+    latestClaim: '0x00',
+    claimed: 138571.8,
+    grantAmount: 250000.0,
+  },
+  {
+    title: 'Optimism Collective',
+    description:
+      'The Optimism Collective is a large-scale experiment in digital democratic governance, built to drive rapid and sustainable growth of the Optimism ecosystem.',
+    date: new Date('2024-09-15'),
+    delegateTo: '0x02',
+    latestClaim: '0x01',
+    claimed: 75000.0,
+    grantAmount: 100000.0,
+  },
+  {
+    title: 'Velodrome Finance',
+    description:
+      'Velodrome is a next-generation AMM built on the Optimism network, designed to maximize capital efficiency and provide sustainable rewards for liquidity providers.',
+    date: new Date('2024-11-30'),
+    delegateTo: '0x03',
+    latestClaim: '0x02',
+    claimed: 50000.0,
+    grantAmount: 200000.0,
+  },
+  {
+    title: 'Synthetix',
+    description:
+      'Synthetix is a decentralized platform on Ethereum for creating and trading synthetic assets. It allows users to gain exposure to a wide range of assets without directly holding them.',
+    date: new Date('2024-08-20'),
+    delegateTo: '0x04',
+    latestClaim: '0x03',
+    claimed: 180000.0,
+    grantAmount: 300000.0,
+  },
+  {
+    title: 'Perpetual Protocol',
+    description:
+      'Perpetual Protocol is a decentralized perpetual contract trading protocol for every asset, empowering traders, liquidity providers & developers to participate in an open financial market with no barriers to entry.',
+    date: new Date('2024-12-10'),
+    delegateTo: '0x05',
+    latestClaim: '0x04',
+    claimed: 90000.0,
+    grantAmount: 150000.0,
+  },
+  {
+    title: 'Lyra Finance',
+    description:
+      'Lyra is an options trading protocol built on Ethereum L2s. It provides a decentralized options trading platform with competitive pricing and deep liquidity.',
+    date: new Date('2025-01-15'),
+    delegateTo: '0x06',
+    latestClaim: '0x05',
+    claimed: 120000.0,
+    grantAmount: 250000.0,
+  },
+  {
+    title: 'Polynomial Protocol',
+    description:
+      'Polynomial is a decentralized options and structured products protocol, enabling users to create, trade, and manage complex financial instruments on-chain.',
+    date: new Date('2024-07-01'),
+    delegateTo: '0x07',
+    latestClaim: '0x06',
+    claimed: 80000.0,
+    grantAmount: 175000.0,
+  },
+];
+
+const Grants = () => {
+  return (
+    <>
+      <div className="flex flex-col gap-6 items-start pt-20 max-w-2xl">
+        <h1 className="text-4xl font-bold">Grants</h1>
+        <p>
+          Explore all the grants and their delegation. Claim tool offer a
```

Comment:
please see the [updated text](https://www.figma.com/design/m3M3dsivGxauagCELcmKJ9/OP--Claim-tool?node-id=45-2966&node-type=&t=lt6uSR8BNeOdFwvk-0) that meg posted

> Explore all grants from the OP Citizen Grants Council and who they've delegated to. For grantees, this claiming tool offers a self-serve interface to claim and delegate your grant.